docs/tutorial.rst

```diff
@@ -737,9 +737,42 @@ make it easier to avoid problems when using Select2ListView. For example:
             widget=autocomplete.ListSelect2(url='country-list-autocomplete')
         )
 
-Since the selections in Select2ListView map directly to a list, there is no
-built-in support for choices in a ChoiceField that do not have the same value
-for every text. ``Select2ListCreateChoiceField`` allows you to provide custom
+By default, the selections in Select2ListView can map directly to a list,
+resulting in the same text and value for each option.
+
+
+To define your own values for each selection, provide a list-of-lists or
+list-of-tuples for the Select2ListView choice_list, and use Select2ListView
+instead of Select2ListView. For example:
+
+.. code-block:: python
+
+    class CountryAutocompleteFromList(autocomplete.Select2ListView):
+    def get_list(self):
```

Comment:
This might be missing an identation level

Reply:
Will fix this and the ci identified issue this evening.

Reply:
Nice, not much to say here, it's really a quality contribution you got right there, with tests and documentation :tophat: